lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -69,6 +69,12 @@ def __init__(
         rho_sample=None,
         rho_bead=1060.0,
     ):
+        if bead_diameter < 1e-6:
```

Comment:
I think that we anchor this a bit closer to real bead size limits: make it 1e-2? I doubt that anyone will be using sub-nanometer beads.

Reply:
Yeah, fair point. I've changed it to 1e-2.